### PR TITLE
cpp: Rename result::raw() to result::release_raw()

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## [6.2.2] - unreleased
 
+- Changed: [[#293](https://github.com/ethereum/evmc/pull/293)]
+  In C++ API `evmc::result::raw()` renamed to `evmc::result::release_raw()`.
```
Comment (axic) [inline CHANGELOG.md]: Is this breaking?

Comment (axic) [reply CHANGELOG.md]: I think it is, but the comment should at least be merged into 6.2

Comment (chfast) [reply CHANGELOG.md]: It's breaking, but 6.2 has broken C++ API anyway. Besides, this method is used internally.

Comment (axic) [reply CHANGELOG.md]: Just checked, `raw` was introduced in 6.2.1 via #257. I don't think renaming it is fixing a bug, the bug was fixed before the renaming, wasn't it?

**Comment by chfast** [comment]
Based on `master`.

**Comment by axic** [comment]
`acquire_raw` ?

**Comment by chfast** [comment]
Renamed to `acquire_raw()`.

**Comment by gumb0** [comment]
But "acquire" is the opposite of what happens. Maybe it could look fine if it were a free function like `res = acquire_raw(evmc_res)`, but `evmc_res.acquire_raw()` look like `evmc_result` acquires something.

**Pull request merged**